### ISSUE TITLE
quarkus-quickstarts/using-openapi-swagger fails to open Swagger docs

### ISSUE THREAD

**Issue by shaneboulden** [quarkusio/quarkus-quickstarts#207]
Steps to reproduce: 
```
git clone https://github.com/quarkusio/quarkus-quickstarts.git
cd quarkus-quickstarts/using-openapi-swagger
mvn compile quarkus:dev
```

Expected results:
Swagger docs are available at http://localhost:8080/swagger-ui

Actual results:
Error message is shown at http://localhost:8080/swagger-ui: https://i.imgur.com/kAbPzXK.png
Navigating to http://localhost:8080/openapi also produces an error: https://i.imgur.com/4yn5dVK.png
The following is shown in the console: 
```
2019-06-04 11:00:30,102 WARN  [io.und.ser.request] (executor-thread-1) UT015022: Requested resource at /quarkus-generated-openapi-doc.JSON does not exist for include method
```

**Comment by cescoffier** [comment]
This is going to be fixed in the 0.16.1 release. Stay tuned, it's going to land very shortly.

**Comment by cescoffier** [comment]
Fixed in 0.16.1.
